app/controllers/admins/ws_panel_controller.rb

```diff
@@ -226,4 +231,19 @@ def edit_driver
     end
   end
 
+  def get_dr_avatar
+    if !current_admin.nil? && current_admin.active == true
+      @driver = Driver.find_for_authentication(id: message[:id])
+      driver_data = { 'id' => @driver.id }
+      if @driver.avatar?
+        driver_data[:thumb] = @driver.avatar.url(:thumb)
+        driver_data[:medium] = @driver.avatar.url(:medium)
+      else
+        driver_data[:thumb] = '/style/no_avatar.png'
```

Comment:
Maybe it's a good idea to save this file in appropriate folders, for example, as `/avatars/original/missing.png`?